lib/sinatra/shopify-sinatra-app.rb

```diff
@@ -189,7 +190,9 @@ def self.registered(app)
         provider :shopify,
                  app.settings.api_key,
                  app.settings.shared_secret,
-
+                 #Fixes for new Shopify callback issues
+                 redirect_uri: app.settings.hostname + "/auth/shopify/callback",
+                 callback_url: app.settings.hostname + "/auth/shopify/callback",
```

Comment:
Sure thing - whats the latest version of the gem? or is there somewhere I
can check quickly?

On Thu, Nov 26, 2015 at 7:05 PM, Kevin Hughes <notifications@github.com>
wrote:

> In lib/sinatra/shopify-sinatra-app.rb
> <https://github.com/kevinhughes27/shopify-sinatra-app/pull/10#discussion_r46002660>
> :
>
> > @@ -189,7 +190,9 @@ def self.registered(app)
> >          provider :shopify,
> >                   app.settings.api_key,
> >                   app.settings.shared_secret,
> > -
> > +                 #Fixes for new Shopify callback issues
> > +                 redirect_uri: app.settings.hostname + "/auth/shopify/callback",
> > +                 callback_url: app.settings.hostname + "/auth/shopify/callback",
>
> can you bump the shopify omniauth gem and then remove the callback_url
> param since it won't be needed. Also can we remove the comment. Then this
> is good to go!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kevinhughes27/shopify-sinatra-app/pull/10/files#r46002660>
> .
>



-- 
Toby Green